### DISCUSSION TITLE
Настройка старта Kodi со страницы плагина.

### DISCUSSION THREAD

**Discussion by Dimca** [quarckster/kodi.kino.pub#191] [Q&A] [answered]
Привет, может кто помочь настроить старт Kodi с отображением содержимого "Я смотрю". Нужна строка в autoexec.ру

**Comment by quarckster** [comment]
Если вам нужен url, то вот он `plugin://video.kino.pub/watching/`

**Comment by Dimca** [comment]
Cпасибо за ответ.
Как тогда прописать всю строку? Я стартую плагин но не попадаю в меню "Я смотрю":   xbmc.executebuiltin("RunAddon(video.kino.pub)")

**Reply by quarckster** [reply]
```python
import xbmc

xbmc.executebuiltin("Container.Update(plugin://video.kino.pub/watching/)")
```

**Comment by Dimca** [comment, marked as answer]
Так работает но только с задержкой : 
import xbmc

xbmc.executebuiltin("RunAddon(video.kino.pub)")
xbmc.sleep(1000)
xbmc.executebuiltin("Container.Update(plugin://video.kino.pub/watching/)")

**Comment by Avalon911** [comment]
Приветствую! Хочу спросить, можно ли в Kodi сделать так чтобы стартовый экран был сразу на плагин кинопаб, а лучше было бы если отображения было бы как на сайте? Как сделать я так и не понял.

**Reply by quarckster** [reply]
> Хочу спросить, можно ли в Kodi сделать так чтобы стартовый экран был сразу на плагин кинопаб

Не уверен, всё-таки Kodi это медиацентр, а не отдельное приложение для кинопаба.

> а лучше было бы если отображения было бы как на сайте

Лучше тогда сразу браузер установить вместо Kodi  и сделать стартовую страницу кинопаб.
